src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ThemeService.cs

```diff
@@ -159,6 +160,27 @@ public async Task<Either<ActionResult, Unit>> DeleteTheme(Guid themeId)
                 });
         }
 
+        public async Task<Either<ActionResult, Unit>> DeleteUITestThemes(CancellationToken cancellationToken = default)
+        {
+            return await _userService.CheckCanManageAllTaxonomy()
+                .OnSuccess(_ => _context.Themes
+                    .Where(theme => theme.Title.Contains("UI test"))
+                    .Include(theme => theme.Topics))
+                .OnSuccessVoid(async themes =>
```

Comment:
I think we need to check `_themeDeletionAllowed` here to prevent people removing themes on prod - the same as in `CheckCanDeleteThemes`.